backend/tracim_backend/lib/core/url_preview.py

```diff
@@ -58,4 +59,5 @@ def get_preview(self, url: str) -> URLPreview:
             )
         except (WebpreviewException, InvalidURL) as exc:
             raise UnavailableURLPreview('Can\'t generate URL preview for "{}"'.format(url)) from exc
-        return URLPreview(title=title, description=description, image=image)
+        image_url = urljoin(url, image_url) if image_url else image_url
```

Comment:
Which falsy values can have `image_url` ? it's look clearer to me to explictly return it instead of relying on variable here.
something like:
```suggestion
        image_url = urljoin(url, image_url) if image_url else None
```